cypress/e2e/managePatches.feature

```diff
@@ -1,6 +1,8 @@
-@common
 @property
-@managePatches
+@authentication
+@common
+@root
+@search
```

Comment:
Does manage patches page use search MFE?
Could I get a link to this page? I never seen it before.